### PR TITLE
Allow authors to extend “observedAttributes”.

### LINKED ISSUE #84: Allow extending `observedAttributes`.
Super minor change, but we expect all `observedAttributes` to come from our statically-declared `properties`. Instead, we should assume that authors may want to observe additional attributes beyond what they declared in the properties block. That is, we should allow this:

```
static get observedAttributes() {
  return [...super.observedAttributes, 'some-other-thing'];
}
```

### PR DESCRIPTION
**Pull request by theengineear**
Closes #84.

### CODE REVIEW

**Comment by theengineear** [comment]
Small (one character) change to allow extension of observedAttributes. I found myself needing this functionality to track attribute changes without needing to define a related property. It essentially let's you go into "manual mode" with certain properties.

**Comment by theengineear** [comment]
FYI @klebba — probably going to merge this in today. I'm gonna look into updating our dependencies today as well.

**Comment by theengineear** [comment]
@klebba — Just bumping this one up for you. Any hesitations here or in the linked ticket?

File: x-element.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ export default class XElement extends HTMLElement {
   /** Extends HTMLElement.prototype.attributeChangedCallback. */
   attributeChangedCallback(attribute, oldValue, value) {
     const { attributeMap } = XElement.__constructors.get(this.constructor);
-    attributeMap.get(attribute).sync(this, value, oldValue);
+    attributeMap.get(attribute)?.sync(this, value, oldValue);
```
Comment (klebba) [inline x-element.js:129]: it would be good to leave a note here about why we're optional chaining here (else it looks less deliberate and more defensive)

**Comment by klebba** [comment]
LG2M — thanks!

**Review by klebba — APPROVED** [review]

File: x-element.js (hunk 1 above)
Comment (theengineear) [reply x-element.js:129]: Yep. Makes sense 👌

**Pull request merged**